### PR TITLE
Adds python package name to AppConfig

### PR DESCRIPTION
**Pull request by bmbouter**
This is a pulpcore 3.20 compatability requirement.

[noissue]

### CODE REVIEW

File: pulp_ansible/app/__init__.py [hunk 1]
```diff
@@ -9,3 +9,4 @@ class PulpAnsiblePluginAppConfig(PulpPluginAppConfig):
     name = "pulp_ansible.app"
     label = "ansible"
     version = "0.14.0.dev"
+    python_package_name = "pulp-python"
```
Comment (gerrod3) [inline pulp_ansible/app/__init__.py:12]: Didn't know this package was also called pulp-python. TIL. :smiley:

Comment (awcrosby) [reply pulp_ansible/app/__init__.py:12]: in https://github.com/ansible/galaxy_ng/pull/1314 we have `python_package_name = "galaxy-ng"` which matches pypi. Why does this not say `pulp-ansible` ... does it refer to https://pypi.org/project/pulp-python/?

Comment (gerrod3) [reply pulp_ansible/app/__init__.py:12]: I'm pretty sure it was a copy-paste error, don't worry you are correct.

Comment (bmbouter) [reply pulp_ansible/app/__init__.py:12]: lol yeah this needs to `pulp-ansible`

**Review by mdellweg — APPROVED** [review]

**Pull request merged**